k8s-kvm/provision_k8s_host_kvm.sh

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env bash
+
+# This script is intended to be run on a SLES box installed as a KVM host.
+# After OS installation, the following commands should also be run to set up directory permissions and the default network (as root):
+# usermod -aG libvirt root
+# virsh net-define net-default.xml
+# virsh net-start default
+# ntpdate time.nist.gov
+# systemctl enable ntpd
+# systemctl start ntpd
+
+set -o errexit
+
+export K8S_VM_NAME=${K8S_VM_NAME:-$1}
+export K8S_VM_IMAGE_NAME=${K8S_VM_IMAGE_NAME:-scf-libvirt-v2.0.6}
+export K8S_VM_IMAGE_PATH=${K8S_VM_IMAGE_PATH:-~/qcow2-disks}
+export K8S_VM_MEM_GIB=${K8S_VM_MEM_GIB:-8}
+
+# Used for displaying external setup information at end of provisioning
+export OBJECT_STORAGE_BASE_URL=${OBJECT_STORAGE_BASE_URL:-\${OBJECT_STORAGE_BASE_URL}}
+export OBJECT_STORAGE_YAML_BUCKET=${OBJECT_STORAGE_YAML_BUCKET:-kube-config}
+
+mkdir -p "$K8S_VM_IMAGE_PATH"
+cd "$K8S_VM_IMAGE_PATH"
+
+if [[ -z $K8S_VM_NAME ]]; then
```

Comment:
Quotes *shouldn't* be needed when using double square brackets (except for maybe with string comparison?), but I'll go ahead and quote anyway for aesthetic purposes